addons/event_booth_sale/models/event_booth_registration.py

```diff
@@ -20,9 +20,6 @@ class EventBoothRegistration(models.Model):
     contact_phone = fields.Char(string='Contact Phone', compute='_compute_contact_phone', readonly=False, store=True)
     contact_mobile = fields.Char(string='Contact Mobile', compute='_compute_contact_mobile', readonly=False, store=True)
 
-    _sql_constraints = [('unique_registration', 'unique(sale_order_line_id, event_booth_id)',
-                         'There can be only one registration for a booth by sale order line')]
-
```

Comment:
Can't we keep this part ? From what I see the changes done here respect this rule or am I missing something ?

Reply:
Hello @jeh-odoo ,
I have made the changes. Can you please have a look ??

Regarding the SQL constraint,
I removed the constraint because it was no longer needed. There will never be a case in our code where this constraint will be violated.
Also, before my fix, this constraint had buggy behaviour. Here is the [LINK](https://www.awesomescreenshot.com/video/22733160?key=9bf9f8410535cd26d6b6702f2bd14cae) to the video showcasing the same.

The reason:
1. We first selected `Booth A1` and `Booth A2`, and then when we hit save, `event_booth_registration` now has 2 records, linking the `event_booth_id` with the `sale_order_line_id`.

2. The second time we edit the values and select `Booth A2` and `Booth A3`, the constraint is triggered, because there is already an existing value for `Booth A2` linked with the current `sale_order_line_id`.

Every time we edit the SO line and select a booth from the `event_booth_configurator`, a new record inside the `event_booth_registration` is created.

Reply:
Hello @shut-odoo 
Thanks for the changes!
As for the constraint I would still keep it, as it's a SQL constraint and it can avoid people to break things by playing in the DB directly.
Cheers!

Reply:
Hello @jeh-odoo...
Done with the changes 👍